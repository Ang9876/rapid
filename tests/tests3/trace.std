T0|w(@1FE[0])|104
T2|r(@1FE[0])|106
T1|r(@1FE[0])|106