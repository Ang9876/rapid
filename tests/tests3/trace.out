1 4NO
1 4NO
1 4NO
1 4NO
1 4NO
1 4NO
1 4NO
1 4NO
1 4NO
1 4YES
1 2NO
1 2NO
1 2YES
1 2YES
1 3NO
1 3NO
1 3NO
1 3NO
1 3NO
1 3YES
1 3YES
